src/kibana/components/doc_table/components/table_row.js

```diff
@@ -104,10 +105,10 @@ define(function (require) {
           }
 
           $scope.columns.forEach(function (column) {
-            newHtmls.push(cellTemplate({
+            newHtmls.push(noWhiteSpace(cellTemplate({
```

Comment:
Removing the `noWhiteSpace` filter here causes this display for the `_source` column:
![screen shot 2015-06-04 at 1 19 35 pm](https://cloud.githubusercontent.com/assets/51061/7993690/c2c40fb6-0abc-11e5-89c6-03200187a52a.png)

Adding the filter back fixes the display:
![screen shot 2015-06-04 at 1 23 13 pm](https://cloud.githubusercontent.com/assets/51061/7993721/eee6e546-0abc-11e5-9032-08989af16bdc.png)

I too think its odd that I need to apply this filter over here. I'm guessing there's another way to fix the `_source` display issue (when the filter is removed) but I haven't been able to figure out how.